docs/conf.py

```diff
@@ -40,7 +40,7 @@
     'sphinx.ext.extlinks',
     'sphinx.ext.napoleon',
     'recommonmark',
-    'sphinxcontrib.spelling',
+    #'sphinxcontrib.spelling',
```

Comment:
why is this commented?

Reply:
oops this is a mistake, will revert back to the original